Regex to extract title from file names => Regex too long

**Describe the bug**
Add too long rexgex

**To Reproduce**
Steps to reproduce the behavior:
1. Go to libraries
2. add  too long Regex:  (?<![\p{L}\p{N}])(Example1|Example2|Example3|Example4|Example5|Example6|Example6|Example7|Example8|Example9|Example10|R\.?G\.?\s*Mechanics|R\.?G\.?\s*Catalyst|R\.?G\.?\s*Games|GOG)(?![\p{L}\p{N}])
3. Save


**Expected behavior**
Caused by: org.h2.jdbc.JdbcSQLDataException: Value too long for column "value CHARACTER VARYING(255)": "'mjP6v23CJGWhfRDWH0xSBjA/RGPAiS5k2lGrvIzCR3d5ZICF15n80KTS8S1tT/44QS6DrYkbzpdyKWL... (428)"; SQL statement:

update app_config set "value"=? where "key"=? [22001-232]

at org.h2.message.DbException.getJdbcSQLException(DbException.java:518)

at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)

at org.h2.message.DbException.get(DbException.java:223)

at org.h2.message.DbException.getValueTooLongException(DbException.java:334)

at org.h2.value.Value.getValueTooLongException(Value.java:2612)

at org.h2.value.Value.convertToVarchar(Value.java:1287)

at org.h2.value.Value.convertTo(Value.java:1143)

at org.h2.value.Value.convertForAssignTo(Value.java:1118)

at org.h2.table.Column.validateConvertUpdateSequence(Column.java:406)

at org.h2.table.Table.convertUpdateRow(Table.java:1005)

at org.h2.command.dml.SetClauseList.prepareUpdate(SetClauseList.java:149)

at org.h2.command.dml.Update.update(Update.java:82)

at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)

at org.h2.command.CommandContainer.update(CommandContainer.java:139)

at org.h2.command.Command.executeUpdate(Command.java:304)

at org.h2.command.Command.executeUpdate(Command.java:248)

at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:213)

at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:172)

at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)

at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)

at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:194)

... 191 common frames omitted

--- a/app/src/main/kotlin/org/gameyfin/app/config/entities/ConfigEntry.kt
+++ b/app/src/main/kotlin/org/gameyfin/app/config/entities/ConfigEntry.kt
@@ -11,6 +11,7 @@ class ConfigEntry(
     @Column(name = "`key`", unique = true)
     val key: String,
 
+    @Lob
     @Column(name = "`value`")
     @Convert(converter = EncryptionConverter::class)
     var value: String

--- a/app/src/main/kotlin/org/gameyfin/app/games/entities/Game.kt
+++ b/app/src/main/kotlin/org/gameyfin/app/games/entities/Game.kt
@@ -40,11 +40,9 @@ class Game(
     var headerImage: Image? = null,
 
     @Lob
-    @Column(columnDefinition = "CLOB")
     var comment: String? = null,
 
     @Lob
-    @Column(columnDefinition = "CLOB")
     var summary: String? = null,
 
     var release: Instant? = null,

--- a/app/src/main/resources/db/migration/V2.2.0.2__Disable_length_limit_for_config_values.sql
+++ b/app/src/main/resources/db/migration/V2.2.0.2__Disable_length_limit_for_config_values.sql
@@ -1,0 +1,8 @@
+-- Flyway Migration: V2.2.0.2
+-- Purpose: Make APP_CONFIG."value" column unbounded to avoid length errors when storing large encrypted values.
+-- Context: Previously defined as CHARACTER VARYING(255); H2 raised 22001 (value too long).
+-- Strategy: Alter column type to CLOB (unlimited length in H2). This matches other large text usages (e.g., COMMENT, SUMMARY) which use CHARACTER LARGE OBJECT.
+
+ALTER TABLE APP_CONFIG
+    ALTER COLUMN "value" CLOB;
+

